plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/Section.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates;
+
+use Automattic\WooCommerce\Admin\BlockTemplates\BlockInterface;
+use Automattic\WooCommerce\Admin\BlockTemplates\ContainerInterface;
+
+class Section extends ProductBlock implements ContainerInterface {
+
+	function __construct( array $config, &$root_template, ContainerInterface &$parent = null ) {
+		parent::__construct( array_merge( array(  'blockName' => 'woocommerce/product-section' ), $config ), $root_template, $parent );
+	}
+
+	/**
+	 * Add a section block type to this template.
+	 *
+	 * @param array $block_config The block data.
+	 */
+	public function add_section( array $block_config ): BlockInterface {
+		$block = new Section( $block_config, $this->get_root_template(), $this );
+		return $this->add_inner_block( $block );
+	}
+
+	/**
+	 * Adds columns to a template.
+	 *
+	 * @param array $column_block_configs Column block data.
+	 */
+	public function add_columns( array ...$column_block_configs ): BlockInterface {
```

Comment:
I don't feel too strongly one way or another about this yet, but same question as above: does this offer us benefit over simply calling `add_block` for the columns and then `add_block` on that?

I assume there's some mental overhead saved in not needing to know the column IDs.  On the other hand, that also means it will be very challenging to add additional blocks to this column at a later point.

Reply:
>  does this offer us benefit over simply calling add_block for the columns and then add_block on that?

Similar reasoning as above, this saves me a lot of boilerplate. I would be more keen on keeping this, given this is more strictly related with the `column(s)`. I could add an option to add a `columns` block id and individual column ids.
What I could do is move this function into the `SimpleProductTemplate` class as a private function, so it feels more like a helper function. We can decide later if we want to move it back to `Section`/`Group` class to make it a default for our templates.

Reply:
I like the idea of saving boilerplate, though still concerned about being able to access this column later without an explicit ID, especially by 3PDs.